src/components/sidebar/SidebarMenu.tsx

```diff
@@ -36,7 +37,21 @@ const SidebarMenu: React.FC<SidebarMenuProps> = ({
         >
             <div css={sidebarMenuDiv} onClick={() => toggle()}>
                 <h5 css={sidebarMenuTitle(theme)}>{title}</h5>
-                <img alt="Arrow" src={imgArrow} css={sidebarMenuDivImg} />
+                {openMenu ? (
+                    <img
+                        alt="Open"
+                        src={imgArrow}
+                        css={sidebarMenuOpenImage}
+                        onClick={() => toggle()}
+                    />
+                ) : (
+                    <img
+                        alt="Close"
+                        src={imgArrow}
+                        css={sidebarMenuCloseImage}
+                        onClick={() => toggle()}
+                    />
+                )}
```

Comment:
so i wouldn't completely replace the component the way that you are with the boolean, rather, you should replace the variables above, something like this:

```
const altText = openMenu ? 'Open' : 'Close';
const sidebarCss = openMenu ? sidebarMenuOpenImage : sidebarMenuCloseImage;
```

then this part would change to:

```suggestion
                  <img
                      alt={altText}
                      src={imgArrow}
                      css={sidebarCss}
                      onClick={toggle}
                  />
             
```

Reply:
love it! too much better! :) thanks for sharing it. I will commit that